app/assets/javascripts/app/core/me.js

```diff
@@ -0,0 +1,37 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('council.core')
+    .service('Me', Me);
+
+  function Me($http, $q) {
+    var me;
+    var mePromise;
+
+    function find() {
+      mePromise = $http.get('/me')
```

Comment:
`.then(...)` also returns a promise so you could avoid the extra variable and return the promise directly